core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5330,7 +5330,6 @@ components:
       description: ""
     CreateRawlsV2BillingProjectFullRequest:
       required:
-        - billingAccount
```

Comment:
If this optional argument is present it should represent a Google billing project, correct? If so, I think it would be good to clarify that in the documentation below (line 5341).

Reply:
I guess it is technically a Rawls billingAccount, but it is only used for GCP?

Reply:
It's a google billing account, and it's now optional given the presence of the managedAppCoordinates arg (technically an `Either` relationship). I'll update the docs 👍